lib/helpers/umzug-helper.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var args = require("yargs").argv;
+var path = require("path");
+var _    = require("lodash");
+
+module.exports = {
+  getStorageMethod: function () {
+    return args.storage || "sequelize";
```

Comment:
Thanks for the tip!  I saw this after I added a test with `fs`.  I will add the config file reading to this PR soon.